app/samples/src/main/java/com/sygic/samples/browsemap/BrowseMapModesActivity.kt

```diff
@@ -26,17 +26,20 @@ package com.sygic.samples.browsemap
 
 import android.os.Bundle
 import android.view.Gravity
+import androidx.appcompat.widget.AppCompatButton
 import androidx.appcompat.widget.PopupMenu
 import com.sygic.maps.module.browsemap.BrowseMapFragment
 import com.sygic.maps.module.common.mapinteraction.MapSelectionMode
 import com.sygic.maps.uikit.viewmodels.common.data.BasicData
 import com.sygic.samples.R
 import com.sygic.samples.app.activities.CommonSampleActivity
 import com.sygic.sdk.map.`object`.MapMarker
-import kotlinx.android.synthetic.main.activity_browsemap_modes.*
+import com.sygic.samples.utils.MapMarkers
 
 class BrowseMapModesActivity : CommonSampleActivity() {
 
+    private val selectionModeButton: AppCompatButton? by lazy { findViewById<AppCompatButton>(R.id.selectionModeButton) }
```

Comment:
toto sa menilo na `nullable`, lebo kotlin extensions vracali `null` za nejakych okolnosti, ci ako to bolo?

Reply:
už si přesně nevzpomínám Míro, ale asi ano.

Reply:
nj, a teda nevies preco si menil z kotlin-ext na lazy , ci?

Reply:
už přesně nevím, mohu to revertnout a uvidíme. ale něco pak nefungovalo. ale možná to ještě souviselo s tím, jak by sample app ve více samostatních modulech / zdrojácích ;) vyzkouším

Reply:
tak po tom refaktoru to už funguje i po staru, revertl jsem tedy ;)